addons/account/views/report_invoice.xml

```diff
@@ -53,17 +53,17 @@
                             <tr>
                                 <!-- TODO: remove in master -->
                                 <t t-set="colspan" t-value="6"/>
-                                <th class="text-left"><span>Description</span></th>
-                                <th class="d-none text-left"><span>Source Document</span></th>
-                                <th class="text-right"><span>Quantity</span></th>
-                                <th t-attf-class="text-right {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}"><span>Unit Price</span></th>
-                                <th t-if="display_discount" t-attf-class="text-right {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}">
-                                    <span>Disc.(%)</span>
```

Comment:
this shouldn't be changed

Reply:
Sorry, a backporting mistake, done. Thanks!